### ISSUE TITLE
Saka options GUI partly non-functional

### ISSUE THREAD

**Issue by grahamperrin** [lusakasa/saka#107]
![2020-03-14 17:16:40 Saka options](https://user-images.githubusercontent.com/192271/76687684-2db35180-661e-11ea-8439-ec6f10c89838.png)

For the second and third preferences: should there be radio buttons, or sliders?

**Comment by pureooze** [comment]
Yes thats correct. Looks like there was bug introduced in the last version when the deps updated. I will have a look and push an update.
